department-of-veterans-affairs/caseflow
# Assign hearings header

Resolves #6876 

### Description
This PR adds the regional office selector to the Assign Hearings homepage.

### Testing Plan
1. Go to /hearings/schedule/assign
1. Confirm there's a regional office selector
1. Confirm that redux is updated when selecting a regional office
